README.md

```diff
@@ -17,43 +17,36 @@ source.
 
 To install this software, you must:
 
-1. Install protocol buffers version 3 or greater. The easiest way is to
-download a release from the [github releases](https://github.com/google/protobuf/releases)
-and add the binary to your `$PATH`. 
-Otherwise [install from source.](https://github.com/google/protobuf/releases)
-2. Of course, install the Go compiler and tools from https://golang.org/.
-See https://golang.org/doc/install for details.
-3. Install the `protoc-gen-go` and `proto` packages for Go. The simplest 
-way is to run 
+1. Install protoc 3 or newer. The easiest way is to
+download a release from [github](https://github.com/google/protobuf/releases)
+and add to `$PATH`.
+Otherwise [install from source.](https://github.com/google/protobuf)
+1. Install the `proto` and `protoc-gen-go` packages. (Later must be in `$PATH`)
```

Comment:
If someone is installing go for the first time I am not sure if they would know that one of those is a binary and one of those is just a library.

Repeating the name is fine. I helped people install go and truss last week, the explanation worked that was there worked perfect. The current instructions would confuse me as I would wonder where the `proto` binary is at.